Bottom: 8cdebe74165c2aec61d78c5ba775217f82881457
Top:    3451ba60de5f0b41830d834bcaa2ae9178f7c796
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 18:13:02 +0000

Refresh of add-flag-parameters-to-resetfo

---

diff --git a/src/action.c b/src/action.c
index 47445d6..9da4c99 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2297,12 +2297,8 @@ ActionConnection (int argc, char **argv, Coord x, Coord y)
 	  {
 	    bool change = false;
 
-	    SaveFindFlag (RATFOUNDFLAG);
-	    change = ResetFoundLinesAndPolygons (true) || change;
-	    RestoreFindFlag ();
-	    SaveFindFlag (FOUNDFLAG);
-	    change = ResetFoundLinesAndPolygons (true) || change;
-	    RestoreFindFlag ();
+	    change = ResetFoundLinesAndPolygons (true, RATFOUNDFLAG) || change;
+	    change = ResetFoundLinesAndPolygons (true, FOUNDFLAG) || change;
 
 	    if (change)
 	      {
